src/types.h

```diff
@@ -35,7 +36,6 @@
 #include <vector>
 
 #ifndef _MSC_VER
-#include <stdint.h>

```

Comment:
The _MSC_VER stuff looks like pre-C++11 support. But I see `-std=c++17` in the build log, so I'd expect that all systems that are able to build Igel also have a working <cstdint>. So, I made this include unconditional and switched to the C++ form.